src/core_plugins/vega/public/vega.less

```diff
@@ -77,3 +77,63 @@
     }
   }
 }
+
+/*
+  Style tooltip popup (gets created dynamically at the top level if dashboard has a Vega vis
+  Adapted from https://github.com/vega/vega-tooltip/blob/master/src/vega-tooltip.css
+ */
+
+.vg-tooltip {
```

Comment:
should match https://elastic.github.io/eui/#/display/tooltip   (or even better, but not sure if possible, use it)